tests/gnrc_udp/main.c

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2015 Freie Universität Berlin
```

Comment:
Well the code is from 2015, I just copied it over from my private app repository. But will extend it to 2017 ;-)